ui/pages/setup_passkey.tsx

```diff
@@ -202,6 +265,48 @@ const SetupPasskey: NextPage = () => {
               )}
             </>
           )}
+          {isSequencedFromSignInFlow && (
+            <>
+              <hr />
+              <div className="u-align--right">
+                <Button
+                  appearance="positive"
+                  type="button"
+                  className="u-no-margin--bottom u-align--right"
+                  onClick={() => {
```

Comment:
inline func, please switch to either a func outside of the component or leverage `useCallback`